test/functional/DDR_Test/src/j9vm/test/ddrext/junit/TestThread.java

```diff
@@ -84,20 +84,6 @@ public void testStackSlots() {
 					Constants.STACKSLOTS_SUCCESS_KEY,
 					Constants.STACKSLOTS_FAILURE_KEY, true));
 		}
-
-		/* test !stackslots thread,sp,a0,pc,literals,els */
-		String paramSet2 = getStackSlotsParamSet(stackSlotsOutput,
-				threadAddress, true);
-		if (paramSet2 == null) {
-			fail("Failed to construct parameter list for command : !stackslots thread,sp,a0,pc,literals,els\nStackSlots Output being used : \n"
-					+ stackSlotsOutput);
-		} else {
-			String stackSlotsOutputParam2 = exec(Constants.STACKSLOTS_CMD,
-					new String[] { paramSet2 });
-			assertTrue(validate(stackSlotsOutputParam2,
-					Constants.STACKSLOTS_SUCCESS_KEY,
-					Constants.STACKSLOTS_FAILURE_KEY, true));
-		}
```

Comment:
Please remove the `needELS` parameter of `getStackSlotsParamSet()` as with this change it will always be `false`.